pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[tool.poetry]
+name = "ha-google-home"
+version = "0.0.0"
```

Comment:
Why do we set version to 0.0.0? Will it have to correspond to package version when we do a release?

Reply:
Otherwise we will need to keep them in sync, it's better when you need to change it only in one place.

Reply:
That's a good point. I'm wondering I go there is GitHub action or some utility to manage the versioning across many files based on gh release number 😅

Reply:
There is a plugin for poetry which can use git tags: https://github.com/mtkennerly/poetry-dynamic-versioning
But Home Assistant components are not regular python packages, so it's not very useful here. You can try it for `glocaltokens` though.

Reply:
Oh perfect, that will be a good place to practice! Thanks man 🤩💥